src/style.css

```
@@ -0,0 +1,78 @@
+@font-face {
+  font-family: Avenir-book;
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir-heavy;
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  color: #3d4e61;
+}
+
+.input {
+  font-family: Avenir-book, Arial, Helvetica, sans-serif;
+  border: none;
+  outline: none;
+  width: 100%;
+  font-size: 16px;
+  color: #3d4e61;
+}
+
+.input_s {
```

better to write the full word
```suggestion
.input_small {
```